README.md

```diff
@@ -189,6 +189,18 @@ Then, visit the site at http://localhost:4001/
 
 [Algolia](https://www.algolia.com) powers our search features. Sometimes after content updates or GTFS releases we will find that the search results do not contain up-to-date results. When this happens you can re-index the Algolia data by running: `mix algolia.update`.
 
+## Commiting Code
+
+When commiting code a bunch of checks are run using [git pre-commit hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks)
+
+This is configured using the tool [husky](https://typicode.github.io/husky/#/) and should install as part of the regular `npm run install` process
+
+This allows all code changed during a commit to be linted and formatted before being commited.
+
+If you wish to modify the checks ran during the pre-commit process the [pre-commit](.husky/pre-commit) file can be modified with any command line commands/scripts
+
+These checks can be skipped by using the `--no-verify` flag when runnint the `commit` git command
+
```

Comment:
question: should devs run the `prepare` npm script that is added in this PR? Or is that not necessary?

Reply:
I believe it is not necessary, I think the during the install of `husky` it looks for that script and runs it.  I know the devs on PickAssist did not need to run it.